invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/CommunityProfileForm.js

```diff
@@ -178,7 +178,27 @@ class CommunityProfileForm extends Component {
         funder: deserializeFunding(fund.funder),
       };
     });
+
     const { customFields } = this.props;
+    /**
+     * Sets the custom vocabulary flag for each item in the provided custom fields.
+     * This flag indicates whether the vocabulary is custom or not.
+     *
+     * @param {object} customFields - The object containing custom fields information.
+     * @param {object} initialValues - The object containing initial values.
+     */
+    ((customFields, initialValues) => {
```

Comment:
minor question: if the flag is set in the configuration of the custom field UI widget, why do we need this step?